src/SAML2/ArtifactResponse.php

```diff
@@ -33,8 +33,6 @@ public function __construct(\DOMElement $xml = null)
 
         if (!is_null($xml)) {
             $status = Utils::xpQuery($xml, './saml_protocol:Status');
-            assert(!empty($status)); /* Will have failed during StatusResponse parsing. */
```

Comment:
In case you're wondering;  this case is already being caught in the parent constructor